From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Wed, 12 May 2021 00:14 +0100
Subject: [PATCH] cloud-init: Disable some modules not relevant to Alpine

Disable some cloud-init modules that are not (currently) relevant
to Alpine and which have been removed from the Alpine package.

Updated 11-Jun-2025 by Jake Buchholz Gokturk <tomalok@gmail.com>
---

diff -aur a/config/cloud.cfg.tmpl b/config/cloud.cfg.tmpl
--- a/config/cloud.cfg.tmpl
+++ b/config/cloud.cfg.tmpl
@@ -203,7 +202,9 @@
   - ansible
   - mcollective
   - salt_minion
+{% if variant not in ["alpine"] %}
   - reset_rmc
+{% endif %}
   - scripts_vendor
   - scripts_per_once
